Automattic/fresh-data
# Add GitHub example

This adds a REST API version of a GitHub API example.

To use local build of fresh-data:

To Test:
1. `cd examples/github-api`
2. `npm install`
3. `cd ../../`
4. `npm run build`
5. `npm run linkLocal`
6. `cd examples/github-api`
7. `npm start`
8. Type in github username, observe results.
